packages/react/src/components/form/form.test.tsx

```diff
@@ -21,6 +27,42 @@ describe("#Form", () => {
     expect(screen.getByTestId("sid-form-initial-state")).toBeInTheDocument();
   });
 
+  test("should render error message - empty email input", async () => {
```

Comment:
Can we add another test case here:
- form is invalid
- we make a change to an input field
- we assert that the error message is removed